lib/Doctrine/DBAL/Schema/Visitor/Graphviz.php

```diff
@@ -41,7 +39,7 @@ public function acceptForeignKey(Table $localTable, ForeignKeyConstraint $fkCons
      */
     public function acceptSchema(Schema $schema)
     {
-        $this->output  = 'digraph "' . sha1(mt_rand()) . '" {' . "\n";
```

Comment:
Should we use the schema FQN for this instead?

Reply:
Not sure what the code would look like. Getting the FQN requires some default namespace.

Besides that, I was unable to generate the chart from the output of the visitor:
```php
$schema = new Schema();
$table  = $schema->createTable('test');
$table->addColumn('foo', 'integer');
$table->addColumn('bar', 'string');

$visitor = new Graphviz();
$visitor->acceptSchema($schema);
$visitor->write('php://stdout');
```
produces the output which doesn't contain any relevant information:
```
digraph {
splines = true;
overlap = false;
outputorder=edgesfirst;
mindist = 0.6;
sep = .2;
}
```

Reply:
Could you provide a snippet for getting the FQN for the chart?

Reply:
Assuming DB "foo" and schema "bar"

`diagraph "foo.bar" {`

or just schema "foo":

`diagraph "foo" {`

Reply:
Let's go with an anonymous `diagraph` block :+1: